src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1205,6 +1205,31 @@ public void TestPlusStringV21()
             TestPlus(context);
         }
 
+        [Test]
+        public void TestPlusNumericBoundary()
+        {
+            TestPlusNumericBoundaryByContext(_contextV20);
+            TestPlusNumericBoundaryByContext(_contextV21);
+            Helper.AssertTemplateResult(expected: "2147483648", template: "{{ '2147483647' | plus: 1 }}", syntax: _contextV21.SyntaxCompatibilityLevel);

```

Comment:
Should we comment that this tests a string which is supported in Syntax 2.1? Should we also test a string as a local variable?